ios/ImagePickerManager.m

```diff
@@ -363,9 +363,13 @@ - (void)imagePickerController:(UIImagePickerController *)picker didFinishPicking
             NSData *data;
             if ([[[self.options objectForKey:@"imageFileType"] stringValue] isEqualToString:@"png"]) {
                 data = UIImagePNGRepresentation(image);
+                NSString *mimeType = (__bridge_transfer NSString *)(UTTypeCopyPreferredTagWithClass(kUTTypePNG, kUTTagClassMIMEType));
+                [self.response setObject:mimeType forKey:@"type"];
             }
             else {
                 data = UIImageJPEGRepresentation(image, [[self.options valueForKey:@"quality"] floatValue]);
+                NSString *mimeType = (__bridge_transfer NSString *)(UTTypeCopyPreferredTagWithClass(kUTTypeJPEG, kUTTagClassMIMEType));
+                [self.response setObject:mimeType forKey:@"type"];
```

Comment:
These two blocks of code only differ in the enum value `kUTTypePNG` vs `kUTTypeJPEG`. Could you assign the value here then set response value out of the if/else blocks?

Reply:
Thanks for the feedback. I pushed a commit that addresses this issue.